src/services/transactions.rs

```diff
@@ -8,15 +8,15 @@ use crate::models::commons::Page;
 use reqwest::Url;
 use anyhow::Result;
 
-pub fn get_about() -> String {
+pub fn get_about() -> Result<String> {
```

Comment:
I will replace the `unwrap` calls with `?` in a different pull request, as we don't need to be so drastic anymore.